extensions/api/identity-api/verifiable-credentials-api/src/main/java/org/eclipse/edc/identityhub/api/verifiablecredentials/v1/unstable/VerifiableCredentialsApiController.java

```diff
@@ -87,7 +87,7 @@ public VerifiableCredentialsApiController(CredentialStore credentialStore,
     @GET
     @Path("/{credentialId}")
     @Override
-    public VerifiableCredentialResource getCredential(@PathParam("credentialId") String id, @Context SecurityContext securityContext) {
+    public VerifiableCredentialResource getCredential(@PathParam("participantContextId") String participantId, @PathParam("credentialId") String id, @Context SecurityContext securityContext) {
```

Comment:
if the `participantId` parameter is needed for the correct generation of Swagger, then it should be added everywhere, e.g. `KeyPairResourceApiController`. 
If it is not needed for that, it should be removed here too.